cirq/linalg/decompositions.py

```diff
@@ -329,8 +329,7 @@ def canonicalize(self, atol: float = 1e-8) -> 'AxisAngleDecomposition':
 
     def _value_equality_values_(self):
         v = self.canonicalize(atol=0)
-        return (value.PeriodicValue(v.angle,
-                                    period=math.pi * 2), v.axis, v.global_phase)
+        return (v.angle % (math.pi * 2), v.axis, v.global_phase)
```

Comment:
This is an incorrect change! It breaks the approximate comparison logic.

Reply:
New cycle avoidance technique makes this irrelevant.